lfs/batcher.go

```diff
@@ -44,29 +51,43 @@ func (b *Batcher) Next() []interface{} {
 	return <-b.batchReady
 }
 
+// Flush causes the current batch to halt accumulation and return
+// immediately, even if it is smaller than the given batch size.
+func (b *Batcher) Flush() {
+	b.flush <- struct{}{}
+}
+
 // Exit stops all batching and allows Next() to return. Calling Add() after
 // calling Exit() will reset the batcher.
 func (b *Batcher) Exit() {
 	atomic.StoreUint32(&b.exited, 1)
 	close(b.input)
+	close(b.flush)
 }
 
 // acceptInput runs in its own goroutine and accepts input from external
-// clients. It fills and dispenses batches in a sequential order: for a batch
-// size N, N items will be processed before a new batch is ready.
+// clients. Without truncation, it fills and dispenses batches in a sequential
```

Comment:
This message still mentions truncation. Can you rewrite it in a way so it calls it "flushing" somehow?